emop-maven-plugin/src/main/java/edu/cornell/VmsMojo.java

```diff
@@ -126,7 +128,7 @@ public void execute() throws MojoExecutionException {
             getLog().info("Number of changed files found: " + offsets.size());
         }
 
-        invokeSurefire();
+        //invokeSurefire();
```

Comment:
Please leave an issue stating that the console output scheme doesn't work and why we think it doesn't!